06-building-a-program-start-to-finish/writing-tests/Output/invertedIndexTest.js

```diff
@@ -0,0 +1,96 @@
+/* Test Setup */
+const myInvertedIndex = new InvertedIndexClass();
+const book = require('./../book.json');
+const bookempty = require('./../bookempty.json');
+const bookwrongformat = require('./../bookwrongformat.json');
+const notBook = require('./../notBook.json');
+const books3 = require('./../books3.json');
+
```

Comment:
Please [see link](https://github.com/airbnb/javascript#naming--filename-matches-export) to understand what is appropriate here